stockflux-securities-master/src/components/Button.css

```diff
@@ -0,0 +1,42 @@
+.button-container.extraSmall > button,
```

Comment:
You're mixing `camelCase` with `kebab-case` here. Can we stick to a single naming convention?

Reply:
yep good point, changed now